sources/packages/forms/src/form-definitions/sfaa2025-26.json

```diff
@@ -10905,6 +11033,50 @@
               "lazyLoad": false,
               "tags": [],
               "hideLabel": true
+            },
+            {
+              "autofocus": false,
+              "input": true,
+              "tableView": false,
+              "inputType": "radio",
+              "label": "Is your parent(s)/step-parent/sponsor/legal guardian a resident of B.C.?",
```

Comment:
In a similar way the BC residency question for student must be asked only for `data.dependantstatus === 'independant'`. This condition was originally at the panel level, and now the panel belongs to residency question for both student and parent. So we need to move the condition to question level instead of panel level.

![image](https://github.com/user-attachments/assets/1a86ac67-2eaa-447f-85be-5a76c7fd4233)

![image](https://github.com/user-attachments/assets/619cc4f7-72a5-4004-bff3-d7fba042db8a)